ipatests/test_ipalib/test_util.py

```diff
@@ -0,0 +1,44 @@
+# Authors:
+#   Oleg Kozlov <okozlov@redhat.com>
+#
+# Copyright (C) 2018  Red Hat
```

Comment:
You copied an old copyright header. IPA switched to a new, shorter copyright header for new files a while ago.